python/tank/platform/bundle.py

```diff
@@ -321,7 +321,7 @@ def import_module(self, module_name):
         from . import framework
         
         # first, set the module we are currently processing
-        framework.CURRENT_BUNDLE_DOING_IMPORT.append(self)
+        framework.set_current_bundle_doing_import(self)
```

Comment:
This confused me a little bit - i think with a stack, we should use the verbs push and pop, this makes the code easier to read I think immediately tells the reader of the code that it's a stack we are dealing with.